src/Frame.js

```diff
@@ -0,0 +1,41 @@
+var Frame = function() {
+  this.frameScore = 0;
+  this.skittlesRemaining = 10;
+};
+
+Frame.prototype.play = function() {
+  score = this.bowl();
+  this.frameScore += score;
+  this.skittlesRemaining -= score;
+  if(this.skittlesRemaining === 0) {
+    return 'Strike';
```

Comment:
Would it not make more sense to set some state as `'Strike'` so it can be accessed later?